charts/osm/values.yaml

```diff
@@ -32,7 +32,8 @@ OpenServiceMesh:
   enablePermissiveTrafficPolicy: false
   enableBackpressureExperimental: false
   enableEgress: false
-  enablePrometheus: true
+  enablePrometheus: false
+  enablePrometheusScraping: false
```

Comment:
Let's keep the default to true to simplify user experience. Even though the scraping is set to on, it will only scrape config if metrics is explicitly enabled on a namespace.